package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -147,10 +147,16 @@ export const renderText = <
     },
   };
 
-  const onLink = (url: string) =>
-    onLinkParams
+  const onLink = (url: string) => {
+    const pattern = new RegExp(/^.+\/\//);
+    if (!pattern.test(url)) {
+      url = 'http://' + url;
+    }
```

Comment:
I might be wrong, but it looks to me like we're doing this check two places - would it make sense to extract it to a function in case we need to change it later? That's if my assumption that the goal is the same for the check here and in `goToURL`

Reply:
Already working on it while writing tests so that we have a global function around it. Good point 😄